authentication/authentication-service/src/main/resources/bootstrap.properties

```diff
@@ -50,3 +50,10 @@ openapi.group.name=${openapi.info.title}
 openapi.group.paths[0]=/**
 springdoc.swagger-ui.disable-swagger-default-url=true
 
+logging.level.root=DEBUG

```

Comment:
Remove ctrl-M chars from this file.

Reply:
Not able to see this ctrl-M character either in local or directly in github https://github.com/mosip/id-authentication/blob/a90ddf10de99ff23c2a494192d39211c907a125b/authentication/authentication-internal-service/src/main/resources/bootstrap.properties

https://github.com/mosip/id-authentication/pull/1346/files Raised 1 sample PR there also it's showing.